Program.cs

```diff
@@ -722,6 +722,97 @@ public static void Props()
             Console.Clear();
         }
 
+        //User Powerup and prep
+        static void PowerUp()
+        {
+            Console.WriteLine("=========================================");
+            Console.WriteLine("==  Successful survived from the dark  ==");
+            Console.WriteLine("=========================================");
+            temp = Console.ReadLine();
```

Comment:
This variable should be initialized as a string. It should also probably be used after the menu display if it is intended to get the users choice.